src/exceptions/fmt-exception.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <exception licenseId="fmt-exception" name="fmt exception">
+   <crossRefs>
+      <crossRef>https://github.com/fmtlib/fmt/blob/master/LICENSE</crossRef>
+      <crossRef>https://github.com/fmtlib/fmt/blob/2eb363297b24cd71a68ccfb20ff755430f17e60f/LICENSE#L22C1-L27C62</crossRef>
+   </crossRefs>
```

Comment:
Here should be:
```
<notes>
  Typically used with MIT.
</notes>

Reply:
that is correct, it was noted in the "license decision" in the issue